products/mssqlserver.md

```diff
@@ -11,135 +11,408 @@ versionCommand: select @@version
 releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
 
 activeSupportColumn: true
+releaseColumn: true
 releaseDateColumn: true
 extendedSupportColumn: Extended Security Updates
 
+# releaseDate/support/eol/extendedSupport: https://learn.microsoft.com/lifecycle/products/?terms=SQL%20Server
+# Overview over all versions: https://learn.microsoft.com/troubleshoot/sql/releases/download-and-install-latest-updates
```

Comment:
I think we can set up an automation with this.

Reply:
I would handle that outside of this PR.